library/src/amd_detail/rocblaslt/src/include/tensile_host.hpp

```diff
@@ -56,6 +56,27 @@ constexpr double value_category(const T& beta)
     return beta == T(0) ? 0.0 : beta == T(1) ? 1.0 : beta == T(-1) ? -1.0 : 2.0;
 }
 
+struct TensileDataGemm
```

Comment:
Exposing this is not recommended.

Reply:
Thanks, the above has been reimplemented in commit ce4ebf5.